lib/airbrakex/notifier.ex

```diff
@@ -21,14 +21,24 @@ defmodule Airbrakex.Notifier do
         |> add_error(error)
         |> add_context(Keyword.get(options, :context))
         |> add(:session, Keyword.get(options, :session))
-        |> add(:params, Keyword.get(options, :params))
+        |> add(:params, filter_parameters(Keyword.get(options, :params), Config.get(:airbrakex, :filter_parameters, [])))
```

Comment:
Cette ligne serait plus claire si on introduisait des variables intermédiaires. De plus, on peut passer un 3e argument à `Keyword.get` pour la valeur par défaut, et ainsi éviter le `(params || %{})` plus bas.

```suggestion
    config = Config.get(:airbrakex, :filter_parameters, [])
    params = Keyword.get(options, :params, %{}) # <--- valeur par défaut ici
    filtered_params = filter_parameters(params, config)
    
    # ...
    
        |> add(:params, filtered_params)
```